### PR TITLE
Apm 1191 use common pipeline

### PR DESCRIPTION
**Pull request by Sourceless**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [x] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: azure/azure-pr-pipeline.yml [hunk 1]
```diff
@@ -1,60 +1,42 @@
 name: "$(SourceBranchName)+$(BuildID)"
 
-# Prevents building every commit or on PR
 trigger: none
 pr: none
```
Comment (vurt007) [inline azure/azure-pr-pipeline.yml:4]: presume this will need a PR trigger at some point ?

File: azure/azure-pr-pipeline.yml [hunk 2]
```diff
@@ -1,60 +1,42 @@
...
 
-# Consume an artifact from the build pipeline as soon as it becomes available.
 resources:
+  repositories:
+    - repository: common
+      type: github
+      name: NHSDigital/api-management-utils
+      ref: refs/heads/apm-1191-common-pipelines
```
Comment (vurt007) [inline azure/azure-pr-pipeline.yml:11]: still got explicit ref to branch

File: azure/azure-pr-pipeline.yml (hunk 1 above)
Comment (strutt) [reply azure/azure-pr-pipeline.yml:4]: Not needed

File: azure/azure-pr-pipeline.yml (hunk 2 above)
Comment (strutt) [reply azure/azure-pr-pipeline.yml]: Removed for all pipelines now utils PR is merged

**Pull request merged**